packages/babel-plugin-jest-hoist/src/index.ts

```diff
@@ -7,16 +7,16 @@
  * @flow
  */
 
-function invariant(condition, message) {
+const invariant = (condition: any, message: any) => {
```

Comment:
why change it to a `const`?

Reply:
I think the code more consistent when all function defined in arrow function in this file.

Reply:
fair 🙂